app/src/main/java/com/teamagam/gimelgimel/app/injectors/modules/ApiModule.java

```diff
@@ -0,0 +1,18 @@
+package com.teamagam.gimelgimel.app.injectors.modules;
+
+import com.teamagam.gimelgimel.data.message.rest.GGMessagingAPI;
+import com.teamagam.gimelgimel.data.message.rest.RestAPI;
+import com.teamagam.gimelgimel.presentation.scopes.PerFragment;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class ApiModule {
+
+    @Provides
+    @PerFragment
```

Comment:
That's what I want to talk about.
I couldn't use singleton because it's different scope.